### ISSUE TITLE
Remove empty lines

### ISSUE THREAD

**Issue by igenaya** [SubtitleEdit/subtitleedit#5034]
I have remove hearing impaired on subtitles and and as the result, the text that removed become empty lines.
i've tried fix common error >remove empty line but it said that there are nothing to fix. what shoud i do?

![Screenshot (91)](https://user-images.githubusercontent.com/83929909/117675627-5366c700-b1df-11eb-9bf1-25ea95025372.png)

**Comment by niksedk** [comment]
Could you attach the subtitle - perhaps both before and after "Remove text for HI".
It's always a big help with the actual subtitle files.

**Comment by igenaya** [comment]
yes  i use that tool

![Screenshot (96)](https://user-images.githubusercontent.com/83929909/117737646-e11cd380-b22c-11eb-973f-9ee4c8d192f5.png)


i've tried for english subtitle and there aren't empty lines
but when i tried removing HI with korean subtitles it goes like this

![Screenshot (97)](https://user-images.githubusercontent.com/83929909/117737671-e9750e80-b22c-11eb-8758-e55643d049ff.png)

**Comment by niksedk** [comment]
Could you attach the subtitle - perhaps both before and after "Remove text for HI"?

**Comment by igenaya** [comment]
[sub.zip](https://github.com/SubtitleEdit/subtitleedit/files/6462527/sub.zip)

**Comment by OmrSi** [comment]
That's because the lines are actually blank, there are hidden Unicode control characters in them.

**Comment by OmrSi** [comment]
The way I see it, right now you have 2 options:
1. Use this in multiple replace: find `^\p{Cf}+$` and replace with nothing as a Regular Expression, which should remove all lines that only contain hidden Unicode control characters.
2. Use the same Regular Expression in normal replace and then use Fix common errors' remove blank lines or the remove blank lines shortcut.

**Comment by niksedk** [comment]
A subtitle file so often helps :)

**Comment by igenaya** [comment]
Thank you soooo much. It worked!!
but I have another problem :)
how to delete this <br thing on the first sentence??
![Screenshot (105)](https://user-images.githubusercontent.com/83929909/118138706-39282580-b439-11eb-8dbe-51746331b20e.jpg)

**Comment by igenaya** [comment]
Thank you so much
I really appreciate it

**Comment by igenaya** [comment]
I've got another problem.
The break between HI text remain and it cause like this.


![Screenshot (108)](https://user-images.githubusercontent.com/83929909/118236830-278e5e80-b4c9-11eb-851c-ff80d969f822.png)

I used tool remove empty line in FCE but it only remove break line at the end.
I tried tool remove break selected line and there's a space left at the beginning text like this. What should I do to fix it?
![Screenshot (109)](https://user-images.githubusercontent.com/83929909/118236841-29f0b880-b4c9-11eb-82a1-729edac93a84.png)

[Copy (3).zip](https://github.com/SubtitleEdit/subtitleedit/files/6477271/Copy.3.zip)
